terraso_backend/apps/story_map/notifications.py

```diff
@@ -21,33 +21,58 @@
 from django.utils import translation
 from django.utils.translation import gettext_lazy as _
 
+from apps.auth.services import JWTService
 from apps.notifications.email import TRACKING_PARAMETERS, EmailNotification
 
 
-def send_memberships_invite_email(memberships, story_map):
-    params = urlencode(TRACKING_PARAMETERS)
-    story_map_url = (
-        f"{settings.WEB_CLIENT_URL}/tools/story-maps/"
-        f"{story_map.story_map_id}/{story_map.slug}/edit?"
-        f"{params}"
-    )
-    users = [
-        membership.user for membership in memberships if membership.user.notifications_enabled()
+def accept_invite_url(user, membership):
+    params = TRACKING_PARAMETERS
+    params["token"] = JWTService().create_story_map_membership_approve_token(membership)
+    return f"{settings.WEB_CLIENT_URL}/tools/story-maps/accept?{urlencode(params)}"
+
+
+def send_memberships_invite_email(inviter, memberships, story_map):
+    signed_up_memberships = [
```

Comment:
question: Does `signed_up_memberships` reflect users who have a Terraso account and `not_signed_up_memberships` users who do not have a Terraso acount?

Reply:
@paulschreiber yes, that is it, those names are clear to me, let me know what are your suggestions for better names